.changeset/twenty-maps-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+feat: Add urls to the first and the last page in the Page interface
```

Comment:
This will need updating, will wait to see what the docs PR is to know what to suggest.